backend/tournesol/views/polls.py

```diff
@@ -161,7 +162,7 @@ def annotate_with_total_score(self, queryset, request, poll: Poll):
                 F("criteria_scores__score") * criteria_weight,
                 filter=Q(criteria_scores__poll=poll),
             )
-        )
+        ).filter(total_score__isnull=False)
```

Comment:
I found it made more sense to have this operation right after the annotation. This means that entities which have no EntityCriteriaScores will not be returned.

Should we also keep a filter on `tournesol_score__isnull=False` ?